runelite-client/src/main/java/net/runelite/client/plugins/lmsperformancetracker/LmsPerformanceTrackerConfig.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2019, Matsyir <https://github.com/Matsyir>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.lmsperformancetracker;
+
+import net.runelite.client.config.Config;
+import net.runelite.client.config.ConfigGroup;
+import net.runelite.client.config.ConfigItem;
+
+@ConfigGroup("lmsPerformanceTracker")
+public interface LmsPerformanceTrackerConfig extends Config
+{
+	@ConfigItem(
+		keyName = "restrictToLms",
+		name = "Restrict to LMS",
+		description = "Restricts use within the LMS area. WARNING: inaccurate outside LMS, as each weapon's combat style has to be manually specified.",
```

Comment:
Indeed it's not fully robust but I believe it should be left an option, since in 90% of cases it will work without issues. Pkers usually stick to very common setups that are part of the LMS gear, and I've also added many extra animations to more widely support this. People also rarely use knives places other than edge/pvp world where they don't use overheads. It seems alright to me to leave the option there given it has a warning, restricted to LMS by default.